.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+# What
+
+- What was done in this Pull Request
+- How was it before
+- Any other information that may be useful for the dev to review Pull Request
+
+## Compatibility
+
+- [ ] Does this change maintain backward compatibility?
```

Comment:
Does any linter change guarantee backward compatibility?

Reply:
Yep, e.g. update of dependencies, extending docs, adding additional features (like other config for another tool)

Reply:
Ah, that's great. Thank you!